docs/concepts/prefetching.md

```diff
@@ -6,3 +6,92 @@ hide_title: true
 ---
 
 # `Prefetching`
+
+The goal of prefetching is to make data fetch _before_ the user navigates to a page or attempts to load some known content.
+
+There are a handful of situations that you may want to do this, but some very common use cases are:
+
+1. User hovers over a navigation element
+2. User hovers over a list element that is a link
+3. User hovers over a next pagination button
+
+### Prefetching with React Hooks
```

Comment:
This whole section (except for the example) misses to mention the fact that there is a hook *and* a callback.
It reads to me as if the hook call were already fetching.